### PR TITLE
Define and activate KVM 11.3.2 release

### PR DESCRIPTION
**Pull request by sslavic**
Towards https://github.com/giantswarm/giantswarm/issues/11478

### CODE REVIEW

**Comment by sslavic** [comment]
@giantswarm/team-ludacris is there kvm-operator release available with calico updated, addressing the CVE?

**Comment by MarcelMue** [comment]
> @giantswarm/team-ludacris is there kvm-operator release available with calico updated, addressing the CVE?

Can you try just setting it again here? We figured out why it wasn't showing correctly on AWS: the version was not passed through correctly when using policy only mode ( AWSCNI / Azure CNI).
It should work on kvm as we do no use policy only mode.

**Comment by sslavic** [comment]
@MarcelMue yes of course, thanks for heads up, will give it a try and share the results.

File: kvm/v11.3.2/release.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - name: flannel-operator
+    version: 0.2.0
+  - name: kvm-operator
+    version: 3.11.1
+  - name: kubernetes
+    version: 1.16.9
+  - name: containerlinux
+    version: 2345.3.1
+  - name: calico
+    version: 3.10.4
```
Comment (sslavic) [inline kvm/v11.3.2/release.yaml:46]: Will cover calico update from 3.10.1 to 3.10.4 in release notes, once testing proves it's actually included by version bump here.

File: kvm/v11.3.2/release.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: v11.3.2
+spec:
+  state: active
+  date: 2020-06-12T12:00:00Z
```
Comment (sslavic) [inline kvm/v11.3.2/release.yaml:7]:
```suggestion
  date: 2020-06-11T15:00:00Z
```

Update release date to today

**Pull request merged**